<ui version="4.0" >
 <class>Schueler_Verwaltung</class>
 <widget class="QDialog" name="Schueler_Verwaltung" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Verwaltung</string>
  </property>
  <widget class="QGroupBox" name="grbPflicht" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>321</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title" >
    <string>Pflichtfelder</string>
   </property>
   <widget class="QLineEdit" name="edtVorname" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>30</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblNachname" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="lineWidth" >
     <number>1</number>
    </property>
    <property name="text" >
     <string>Nachname:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblVorname" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Vorname:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtNachname" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>80</x>
      <y>60</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnOK" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>590</x>
     <y>360</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>OK Anzeige</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="grbKommentar" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>110</y>
     <width>321</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title" >
    <string>Kommentar</string>
   </property>
   <widget class="QTextEdit" name="textEdit" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>301</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="grbRechte" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>260</y>
     <width>691</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title" >
    <string>Rechte</string>
   </property>
   <widget class="QRadioButton" name="rbtnSchueler" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>20</y>
      <width>92</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Schüler</string>
    </property>
    <property name="checked" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtnOrganisator" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>50</y>
      <width>92</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Organisator</string>
    </property>
    <property name="checked" >
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtPasswort" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>450</x>
      <y>20</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtWiederholen" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>450</x>
      <y>50</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblPasswort" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>370</x>
      <y>20</y>
      <width>54</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Passwort:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblWiederholen" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>370</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text" >
     <string>Passwort
wiederholen:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnVerlauf" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>360</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Verlauf</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAktion" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>300</x>
     <y>360</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Übernehmen Neu</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="grbOptional" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>350</x>
     <y>10</y>
     <width>351</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title" >
    <string>optionale Felder</string>
   </property>
   <widget class="QLabel" name="lblNeuerNachname" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>30</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>neuer Nachname:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblStrHnr" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>60</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Straße und HNr.:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblOrt" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>120</y>
      <width>54</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Wohnort:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPLZ" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>90</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Postleitzahl:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLand" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>150</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Land:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblEmail" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>210</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>E-Mail-Adresse:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtNeuerNachname" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>30</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtStrHnr" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>60</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtPLZ" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>90</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="layoutDirection" >
     <enum>Qt::LeftToRight</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtOrt" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>120</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtLand" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>150</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtTelefon" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>180</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="edtEmail" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>110</x>
      <y>210</y>
      <width>231</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblTelefon" >
    <property name="enabled" >
     <bool>true</bool>
    </property>
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>180</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text" >
     <string>Telefonnummer:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnBearbeiten" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>360</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Bearbeiten</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="grbVerlauf" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>390</y>
     <width>691</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title" >
    <string>Verlauf</string>
   </property>
   <widget class="QTreeWidget" name="treeWidget" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>20</y>
      <width>671</width>
      <height>241</height>
     </rect>
    </property>
    <column>
     <property name="text" >
      <string>Datum</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Organisator</string>
     </property>
    </column>
    <column>
     <property name="text" >
      <string>Änderungen</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnAbbrechen" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="geometry" >
    <rect>
     <x>440</x>
     <y>360</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Abbrechen</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
